diff/lcow/lcow.go

```diff
@@ -60,10 +60,7 @@ func init() {
 				return nil, err
 			}
 
-			ic.Meta.Platforms = append(ic.Meta.Platforms, ocispec.Platform{
-				OS:           "linux",
-				Architecture: runtime.GOARCH,
-			})
+			ic.Meta.Platforms = append(ic.Meta.Platforms, platforms.DefaultSpec())
```

Comment:
Unfortunately this was done for a reason. LCOW is a project aimed at running Linux containers on a Windows host by using a lightweight VM. What that means for this at least is the containerd bits are running on windows so we'd end up invoking https://github.com/containerd/containerd/blob/c0bbaf5d15bdfa39fe74b9d46b5e2c494ebd388d/platforms/defaults_windows.go#L30-L39.

Reply:
:laughing::laughing:
Let 's just close the pr...